.pre-commit-config.yaml

```
@@ -28,3 +36,11 @@ repos:
     rev: v3.0.3
     hooks:
       - id: prettier
+  - repo: https://github.com/commitizen-tools/commitizen
+    rev: v3.15.0 # automatically updated by Commitizen
+    hooks:
+      - id: commitizen
+      - id: commitizen-branch
+        args: [--rev-range, $PRE_COMMIT_FROM_REF..$PRE_COMMIT_TO_REF]
+        stages:
+          - post-checkout
```

```suggestion
          - pre-push
```
post-commit没有ref的输入，所以跳过